python/app/qt_importer.py

```diff
@@ -68,10 +60,5 @@ def _move_attributes(dst, src, names):
 
 
 # Handle QRegExp / QRegularExpression
-if hasattr(QtCore, "QRegularExpression"):
-    qt_re_module = QtCore.QRegularExpression
-    qt_re_module_is_regular_expression = True
-else:
-    # NOTE remove once Qt4 support is removed
-    qt_re_module = QtCore.QRegExp
-    qt_re_module_is_regular_expression = False
+qt_re_module = QtCore.QRegularExpression
+qt_re_module_is_regular_expression = True
```

Comment:
Unfortunately, this requires more work on python/app/syntax_highlighter.py.

You can take a look at #25. Probably it's worth removing dead code such as `match_multiline_regexp` and `highlight_block_regexp`.